libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -57,38 +61,24 @@
 import java.io.IOException;
 import java.net.HttpURLConnection;
 import java.net.URI;
-import java.util.Calendar;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 /**
- * This class houses functionality related to push notifications.
- * It performs registration and unregistration of push notifications
- * against the Salesforce connected app endpoint. It also receives
- * push notifications sent by the org to the registered user/device.
+ * Provides a default implementation of push notifications registration and
+ * receipt features using the Salesforce connected app endpoint.
  *
  * @author bhariharan
  * @author ktanna
- *
- * @deprecated This class will be drastically altered or replaced in Mobile SDK 11.0 when the deprecated
- * {@link androidx.core.app.JobIntentService} base is replaced with a WorkManager Worker class.
+ * @author Eric C. Johnson <Johnson.Eric@Salesforce.com>
  */
-@Deprecated
-public class PushService extends JobIntentService {
+public class PushService {
```

Comment:
A crux change is this type no longer extends the deprecated `JobIntentService`.  However, it's a public type and one existing users may have subclassed.  So, it lives on here in modified form with hopefully no or minimal breaking changes.

Reply:
@wmathurin Do we think any apps have extended this?  At a glance SFS never has and Salesforce app used to have an `EncryptedPushService` but they deleted it when we added that feature.  I doubt external apps have.  I am not sure there is much value in making the class internal though.